i18n/README.id.md

```diff
@@ -0,0 +1,91 @@
+<p align="center">
+<img width="300" src="https://raw.githubusercontent.com/supabase/supabase/master/web/static/supabase-light-with-background.svg"/>
+</p>
+
+---
+
+# Supabase
+
+[Supabase](https://supabase.io) adalah sumber terbuka alternatif Firebase. Kita mengembangkan fitur Firebase menggunakan alat sumber terbuka kelas perusahaan.
+
+- [x] Menggunakan database Postgres yang di host
+- [x] Langganan waktu sebenarnya
+- [x] Otentikasi dan otorisasi
+- [x] API yang dibuat secara otomatis
+- [x] Dashboard
```

Comment:
You can use `Dasbor` to replace `Dashboard`. But you can leave it as is, it's fine already.